front/app/modules/commercial/content_builder/admin/components/CraftComponents/TwoColumn/index.tsx

```diff
@@ -22,18 +35,98 @@ const TwoColumn: UserComponent = () => {
       w="100%"
       gap="4px"
     >
-      <Box flex="1">
+      <Box
+        flex={columnLayout === '1-1' ? '1' : columnLayout === '1-2' ? '1' : '2'}
+      >
         <Element id="column1" is={Container} canvas />
       </Box>
-      <Box flex="1">
+      <Box
+        flex={columnLayout === '1-1' ? '1' : columnLayout === '1-2' ? '2' : '1'}
+      >
         <Element id="column2" is={Container} canvas />
       </Box>
     </Box>
   );
 };
 
+const TwoColumnSettings = () => {
+  const {
+    actions: { setProp },
+    columnLayout,
+  } = useNode((node) => ({
+    columnLayout: node.data.props.columnLayout,
+  }));
+
+  return (
+    <Box mt="-30px" mb="30px">
```

Comment:
Let's avoid negative margins where we can.

Reply:
@IvaKop Since the titles here have a lot of padding, it ends up looking like this without negative margins (in the prototype, they should be much closer together):
![margin](https://user-images.githubusercontent.com/33987955/163991273-993712ad-a123-487a-80a4-ca93065fb2e3.png)
What would you recommend to reduce the vertical spacing in a case like this?